docs/source/docs/installation/installation_dev.rst

```diff
@@ -54,10 +54,11 @@ installation directory called ``megaqc.db``
 5. Start megaqc
 ---------------
 
-Start MegaQC:
+Start MegaQC. And you need to use poetry shell before running it.
```

Comment:
Explain that you need to use `poetry shell` before running *any megaqc command*.